thoth/package_analyzer/cli.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# thoth-digests-fetcher
+# thoth-package-analyzer
 # Copyright(C) 2019 Fridolin Pokorny
```

Comment:
Append your name to files you have modified as developer of package-analyzer :wink: